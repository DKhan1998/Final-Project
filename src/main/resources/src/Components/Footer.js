import React from "react";
import {Navbar, Container, Col} from "react-bootstrap";

const Footer = () => {
    let fullyear = new Date().getFullYear();
    return (
        <div>
            <Navbar fixed="bottom" bg="dark" variant="dark">
                <Container>
                    <Col lg={12} className="text-center text-muted">
                        <div>
                            &copy; {fullyear}-{fullyear+1} Team 1 All Rights Reserved
                        </div>
                    </Col>
                </Container>
            </Navbar>
        </div>
    )
}

export default Footer;