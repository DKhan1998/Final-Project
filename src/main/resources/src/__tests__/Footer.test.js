import React from "react";
import Footer from "../Components/Footer"
import render from "react-test-renderer";

//snapshot test
test("it matches the snapshot", () => {
    const tree = render.create(<Footer/>).toJSON();
    expect(tree).toMatchSnapshot();
})