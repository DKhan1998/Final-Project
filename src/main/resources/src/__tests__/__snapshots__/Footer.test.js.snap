// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches the snapshot 1`] = `
<div>
  <nav
    className="navbar navbar-expand navbar-dark bg-dark fixed-bottom"
  >
    <div
      className="container"
    >
      <div
        className="text-center text-muted col-lg-12"
      >
        <div>
          Â© 
          2020
          -
          2021
           Team 1 All Rights Reserved
        </div>
      </div>
    </div>
  </nav>
</div>
`;
